StorySquad/Model/Submission.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  Submission.swift
+//  StorySquad
+//
+//  Created by Bohdan Tkachenko on 6/7/21.
+//  Copyright © 2021 Spencer Curtis. All rights reserved.
+//
+
+import Foundation
+
+class Submission: Codable {
+    
+    enum CodingKeys: String, CodingKey {
+        case id = "ID"
+        case childId = "ChildID"
+        case storyId = "StoryID"
+        case cohortId = "CohortID"
+        case hasRead = "HasRead"
+        case hasWritten = "HasWritten"
+        case hasDrawn = "HasDrawn"
+        case complexity = "Complexity"
+        case status = "Status"
+        case pages
+        case image
+    }
+    
+    var id: Int
+    var childId: Int
+    var storyId: Int
+    var cohortId: Int
+    var hasRead: Bool
+    var hasWritten: Bool
+    var hasDrawn: Bool
+    var complexity: String?
+    var pages: [Page]
+    var image: Image
+    
```

Comment:
Consider making image optional, making pages optional or a default of an empty array, and then changing the decoder to use decodeIfPresent for those two properties.